package.json

```diff
@@ -15,6 +18,14 @@
   "license": "MIT",
   "devDependencies": {
     "concurrently": "^8.2.0",
+    "eslint-config-prettier": "^8.8.0",
+    "husky": "^8.0.3",
+    "lint-staged": "^13.2.2",
+    "prettier": "^2.8.8",
     "typescript": "^5.1.3"
+  },
+  "lint-staged": {
+    "*.{ts,js}": "eslint --fix",
+    "*": "prettier --write --ignore-unknown"
```

Comment:
I want to run `prettier` on any file it supports (a lot more than just code: markdown, json, yaml), but since this`*` glob overlaps with the `*.{ts,js}` glob above, we have to [turn off concurrency for lint-staged](https://github.com/karlwernsman/DougDnD/pull/3/files#diff-d2bc4bbf14eadc292d84e0ef5f7a93115c23557f533809fc80b896934291529dR4) or else eslint + prettier potentially try to write to the file at the same time :( Which is no fun but seems easier than the alternatives.

https://github.com/okonet/lint-staged#task-concurrency